atom/common/api/resources/context_menus_bindings.js

```diff
@@ -2,13 +2,43 @@ var ipc = require('ipc_utils');
 var process = requireNative('process');
 var extensionId = process.GetExtensionId();
 
+var id = 0
+
+var createMenuItemId = () => {
+  return 'context-menus-' + extensionId + id++
+}
+
 var binding = {
-  removeAll: function () {
-    console.warn('chrome.contentMenus.removeAll is not supported yet')
+  onClicked: {
+    addListener: function (cb) {
+      ipc.on('chrome-context-menus-clicked', function(evt, info, tab) {
+        cb(info, tab)
+      })
+    },
+    removeListener: function (cb) {
+      // TODO
+      // ipc.off('chrome-context-menus-clicked', cb)
+    }
+  },
+  removeAll: function (cb) {
+    var responseId = ++id
```

Comment:
can we add the extension id to this as well? I know none of the others have it, but we should probably add for all of them because there is a chance of conflict now that we have more than one extension running at the same time